specification/gedcom-03-datamodel.md

```diff
@@ -146,10 +146,12 @@ n HEAD                                     {1:1}
      +2 DATA <Text>                        {0:1}  g7:HEAD-SOUR-DATA
         +3 DATE <DateExact>                {0:1}  g7:DATE-exact
            +4 TIME <Time>                  {0:1}  g7:TIME
+              +5 PHRASE <Text>             {0:1}  g7:PHRASE
```

Comment:
Do we really need a time phrase under a `<DateExact>`?  Why can't we put it only under a `<DateValue>`?

Reply:
Because both are `g7:TIME` structures, and substructures are defined by structure type alone

Reply:
Discussed 2022-11-01
We should use a different URI for this TIME